pensieve/cli.py

```diff
@@ -54,7 +59,7 @@ def run(project_id, dataset_id, date, dry_run):
     active_experiments = collection.end_on_or_after(date).of_type(("pref", "addon"))
 
     # Create a trivial configuration containing defaults
-    spec = AnalysisSpec.from_dict({})
+    spec = AnalysisSpec.from_dict(toml.load(DEFAULT_METRICS_CONFIG))
```

Comment:
When we support experiment-specific configuration, the next step will be to call `spec.merge(AnalysisSpec.from_dict(experiment_specific_configuration))`, right?

Reply:
Correct.